benchmark/web/dart/laska-0.1-aot-3.4/pubspec.yaml

```diff
@@ -0,0 +1,8 @@
+name: benchmark
+
+environment:
+  sdk: ">=3.4.0"
+
+dependencies:
+  laska:
+    git: https://github.com/tenderowl/laska.git
```

Comment:
Could I point a commit or you think it is better to put a tag here?

Reply:
As long as your library doesn't have high traffic, Github alone is not enough. A library should be serious enough so that developers might want to use it - that's why I created sharkbench.dev - to help developers find libraries they might want to use for their next project. I would personally not use your project because it's only on Github and not on pub.dev, so you could take down your repo at any moment.

Reply:
Fair enough! I'll come later with a public release 🪿

Reply:
Looking forward for your public release!